spec/acceptance/class_logstash_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 shared_examples 'a logstash installer' do
   it "installs logstash version #{LS_VERSION}" do
-    expect(shell('/usr/share/logstash/bin/logstash --version').stdout).to eq("logstash #{LS_VERSION}\n")
+    expect(shell('/usr/share/logstash/bin/logstash --version').stdout).to contain("logstash #{LS_VERSION}")
```

Comment:
This test is actually a bit tricky to have in shared_examles since it is used in examples that install the latest version available in the yum repo, and it will fail whenever elastic releases a new patch release ...
I had to update the default LS_VERSION in helpers.rb to make the test pass.